### PR TITLE
feat: added unit tests for header

### PR DESCRIPTION
**Pull request by baranero**

### CODE REVIEW

**Comment by baranero** [comment]
I had an issues with commit. I changed husky configuration. Commit works if you will do it like this way:
`npm run format`
`git add .`
`git commit -m "<type>: <subject>"`
`gir push origin <branch-name>`

File: app/components/Header/Header.tsx [hunk 1]
```diff
@@ -33,6 +34,7 @@ const Header = () => {
 						/>
 					) : (
 						<BsSunFill
+							aria-label="sun"
```
Comment (lausuarez02) [inline app/components/Header/Header.tsx:37]: I would add formatting here but for now it's good enough

File: package.json [hunk 2]
```diff
@@ -13,6 +19,7 @@
 		"test:e2e": "playwright test",
 		"prepare": "husky install",
 		"prettier": "prettier \"**/*.+(js|json|ts|jsx|tsx)\"",
+		"prettier-check": "prettier --check \"**/*.+(js|json|ts|jsx|tsx)\"",
```
Comment (lausuarez02) [inline package.json:22]: I do not understand how you add the prettier-check library but it doesnt have been formatting

File: app/components/Header/Header.tsx (hunk 1 above)
Comment (lausuarez02) [reply app/components/Header/Header.tsx:37]: Fomat this please. :)

**Review by lausuarez02 — CHANGES_REQUESTED** [review]

**Review by kirandash — APPROVED** [review]
LGTM

**Pull request merged**